packages/docs/package.json

```diff
@@ -5,9 +5,9 @@
   "author": "John Leider <john@vuetifyjs.com>",
   "version": "2.2.12",
   "scripts": {
-    "dev": "cross-env node --max_old_space_size=4096 server",
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --config build/webpack.config.js --hot --progress",
     "start": "cross-env NODE_ENV=production node server",
-    "build": "rimraf dist && cross-env NODE_ENV=production webpack --config build/webpack.prod.config.js --no-progress --hide-modules",
+    "build": "rimraf dist && cross-env NODE_ENV=production webpack --config build/webpack.config.js --no-progress --hide-modules",
```

Comment:
You will need to configure the correct output directory in the [project dashboard](https://zeit.co/blog/advanced-project-settings)  or you can add `mv packages/docs/dist public` to move the build output into the `public` directory.